aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,134 @@
+uid: blazor/hybrid/tutorials/maui
+---
+# Build a .NET MAUI Blazor app
+
+This tutorial shows you how to build and modify a .NET MAUI Blazor app. You learn how to:
+
+> [!div class="checklist"]
+> * Create a .NET MAUI Blazor app project
+> * Run the project in the Windows Machine emulator
+> * Run the project in the Android emulator
+
+At the end of this tutorial, you'll have a working app.
+
+## Prerequisites
+
+* [Supported platforms (.NET MAUI documentation)](/dotnet/maui/supported-platforms)
+* [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/) with the following workloads:
+  * **Mobile development with .NET**
+  * **Universal Windows Platform development**
+* [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/): WebView2 is required on Windows when running a native app. When developing .NET MAUI Blazor apps and only running them in Visual Studio's emulators, WebView2 isn't required.
+
+## Installation
+
+Follow the guidance in [Installation (.NET MAUI documentation)](/dotnet/maui/get-started/installation).
+
+## Create a .NET MAUI Blazor app
+
+Launch Visual Studio 2022 17.1 Preview, and in the start window click **Create a new project** to create a new project:
+
+:::image type="content" source="maui/_static/new-solution.png" alt-text="New solution.":::
+
+In the **Create a new project** window, select **MAUI** in the **Project type** drop-down, select the **.NET MAUI App (Preview)** template, and click the **Next** button:
+
+:::image type="content" source="maui/_static/new-project.png" alt-text="Choose a template.":::
+
+> [!NOTE]
+> If the **.NET MAUI App (Preview)** project template does not appear in the **Create a new project** window, open the Developer Command Prompt for VS 2022 Preview and run the following command: `devenv.exe /updateConfiguration`. Then, restart Visual Studio.
+
+In the **Configure your new project** window, name your project, choose a suitable location for it, and click the **Create** button:
+
+:::image type="content" source="maui/_static/configure-project.png" alt-text="Configure the project.":::
+
+Wait for Visual Studio to create the project and for the project's dependencies to be restored:
+
+:::image type="content" source="maui/_static/restored-dependencies.png" alt-text="Restored dependencies.":::
+
+In the **Android SDK License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/android-sdk-license.png" alt-text="Android SDK License Acceptance window.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+Wait for Visual Studio to download the Android SDK and Android Emulator.
+
+## Run the app in the Windows Machine Emulator
+
+In the Visual Studio toolbar, select the **Windows Machine** button to build the app:
```

Comment:
Initially the run button will say "Android Emulator", not "Windows Machine". I'm pushing to change this default, but currently we have to instruct the user to click the little down arrow on the run button and select the "Windows Machine" launch profile instead.

Reply:
Since we're going to have to tell the user how to run on Android anyway, we might as well put the instructions for running on Android first, at least until we can get the default target platform changed.